crates/bevy_ecs/src/relationship/related_methods.rs

```diff
@@ -13,8 +13,17 @@ use core::{marker::PhantomData, mem};
 use super::OrderedRelationshipSourceCollection;
 
 impl<'w> EntityWorldMut<'w> {
+    /// Spawns a entity related to this entity (with the `R` relationship) by taking a bundle
+    pub fn with_related<R: Relationship>(&mut self, bundle: impl Bundle) -> &mut Self {
+        let parent = self.id();
+        self.world_scope(|world| {
+            world.spawn((bundle, R::from(parent)));
+        });
+        self
+    }
+
     /// Spawns entities related to this entity (with the `R` relationship) by taking a function that operates on a [`RelatedSpawner`].
-    pub fn with_related<R: Relationship>(
+    pub fn with_relationships<R: Relationship>(
```

Comment:
I don't like the plural form here either.

Reply:
Yeah `with_relationships` reads to me as "add these Relationships to the current entity" which is the opposite of what is happening.